pythonforandroid/recipes/lxml/__init__.py

```diff
@@ -0,0 +1,45 @@
+from pythonforandroid.toolchain import Recipe, shutil
+from pythonforandroid.recipe import CompiledComponentsPythonRecipe
+from os.path import exists, join, dirname
+
+
+class LXMLRecipe(CompiledComponentsPythonRecipe):
+    version = "3.6.0"
+    url = "https://pypi.python.org/packages/source/l/lxml/lxml-{version}.tar.gz"
+    depends = ["python2", "libxml2", "libxslt"]
```

Comment:
So you have only tested it under `python2`?

Reply:
yes, my project is not yet Python 3 compatible.

Reply:
OK so it means, I either need to test it by hand, or try to make conditional builds also working with `python2` :smile: